.gitignore

```diff
@@ -1,12 +0,0 @@
-.sconf_temp
```

Comment:
Are you sure that there are no files to be ignored after a build completes?

Reply:
This now relies on the `eupspkg` default logic for when a repo has nothing in it, which is to do nothing.  And I've (now) checked that `eupspkg -e build` does not produce any files.

Reply:
I *think* `lsst-build` will make a `_build.log` file.

Reply:
I'll add that just in case.